packages/engine/src/scene/functions/GLTFConversion.ts

```diff
@@ -153,6 +153,7 @@ export const sceneToGLTF = (roots: Object3DWithEntity[]) => {
  * @param mode 'encode' or 'decode'
  */
 const handleScenePaths = (gltf: any, mode: 'encode' | 'decode') => {
+  if (Engine.instance.publicPath == undefined) return
```

Comment:
the public path should always be defined (needs to be manually defined in tests that rely on it) rather than aborting